marklogic-data-hub-central/ui/src/components/queries/managing/manage-query-modal/manage-query.tsx

```diff
@@ -32,6 +35,10 @@ const QueryModal = (props) => {
         getQueries();
     }, [mainModalVisibility, editModalVisibility, deleteModalVisibility]);
 
+    useEffect(() => {
```

Comment:
I don't think you need this useEffect. Call getPreview() in displayExportModal() function